### ISSUE TITLE
train

### ISSUE THREAD

**Issue by haoyuee** [GuoShi28/CBDNet#25]
how do you get your training model ,I use the training code of DnCNN in my own dataset ,there is some trouble in my training.How to solve it?

![image](https://user-images.githubusercontent.com/52694395/60869040-d247ed00-a260-11e9-86c1-87c28cd4e558.png)

**Comment by GuoShi28** [comment]
Hi, the error means that there no layer name "objective1" in the network structure. You should check your own network layer define. "objective1" layer is the loss function layer in my structure and it is removed in the testing phase which may not the same with your own network.

**Comment by haoyuee** [comment]
thanks

**Comment by haoyuee** [comment]
When I run the training code,I face a new problem.![image](https://user-images.githubusercontent.com/52694395/61709743-7615ba80-ad82-11e9-8eb3-be3433035942.png)How to solve it?Thanks!

**Comment by GuoShi28** [comment]
I also met this problem before. I remember that it may be caused by the configuring problem of Matconvnet. I recommend checking the Matconvnet configuring first. If there's still happen this problem, please feel free to contact me.
